python/ray/data/_internal/iterator/iterator_impl.py

```diff
@@ -58,4 +58,6 @@ def __getattr__(self, name):
         raise AttributeError()
 
     def _get_dataset_tag(self):
-        return (self._base_dataset._plan._dataset_name or "") + self._base_dataset._uuid
+        return (
+            self._base_dataset._plan._dataset_name or "dataset"
+        ) + self._base_dataset._uuid
```

Comment:
I forgot to update this part back when we started using statsactor to generate dataset uuids.